frontend/tests/playwright/product/add_to_cart.spec.ts

```diff
@@ -137,7 +139,18 @@ test.describe.serial('product page add to cart', () => {
       });
 
       test('Out of stock product cannot be added to cart', async ({ page }) => {
-         await page.goto('/products/iphone-6s-plus-replacement-battery-out-of-stock');
+         await page.goto(
+            '/products/iphone-6s-plus-replacement-battery-out-of-stock'
+         );
+         await expect(
+            page.getByRole('img', { name: 'Fix Kit' })
+         ).not.toBeVisible();
+         await expect(
+            page.getByRole('img', { name: 'Part Only' }).first()
```

Comment:
Why the `first()` call? Shouldn't there only be one?

Reply:
I was following the same assertions as https://github.com/iFixit/react-commerce/blob/587bdf75a46320f6a4d48e7e8ab7df791537f2ba/frontend/tests/playwright/product/fix_kit.spec.ts#L26-L41

I think it's possible to match the images in `featured products`

Reply:
I _think_ `first()` is called by default. I could be wrong.

Reply:
I am not sure if that is a `Jest` or `Cypress` default, but `Playwright` will throw an error https://playwright.dev/docs/locators#throws-an-error-if-more-than-one